server/routes/referrals/completionDeadlinePresenter.test.ts

```diff
@@ -72,19 +83,19 @@ describe('CompletionDeadlinePresenter', () => {
 
   describe('title', () => {
     it('returns a title', () => {
-      const referral = draftReferralFactory
-        .serviceCategorySelected()
-        .build({ serviceCategory: { name: 'social inclusion' } })
-      const presenter = new CompletionDeadlinePresenter(referral)
+      const serviceCategory = serviceCategoryFactory.build()
+      const referral = draftReferralFactory.serviceCategorySelected(serviceCategory.id).build()
+      const presenter = new CompletionDeadlinePresenter(referral, serviceCategory)
 
-      expect(presenter.title).toEqual('What date does the social inclusion service need to be completed by?')
```

Comment:
I think that it's good practice not to rely on the values which are set in the factory. The idea of the factory, as I understand it, is to provide you with a valid, realistic object, possibly in a predefined state. But the actual values inside are subject to change. So I think it's better to write this as

```typescript
const serviceCategory = serviceCategoryFactory.build(name: "social inclusion")
```

and keep the assertion as it was.